### PR TITLE
qa/suites/upgrade/hammer-jewel-x: add luminous.yaml

### PR DESCRIPTION
**Pull request by tchaikov**
to setup mgr and set the require-osd-release bit in osdmap

Fixes: http://tracker.ceph.com/issues/20342
Signed-off-by: Kefu Chai <kchai@redhat.com>

### CODE REVIEW

**Comment by tchaikov** [comment]
being tested at http://pulpito.ceph.com/kchai-2017-06-20_02:44:54-upgrade-master---basic-mira/

**Comment by tchaikov** [comment]
being tested at 

- http://pulpito.ceph.com/kchai-2017-06-20_06:15:05-upgrade-master---basic-mira/
- http://pulpito.ceph.com/kchai-2017-06-20_06:14:30-upgrade-master---basic-mira/

**Comment by tchaikov** [comment]
being tested at 

- http://pulpito.ceph.com/kchai-2017-06-20_08:51:09-upgrade-master---basic-mira/
- http://pulpito.ceph.com/kchai-2017-06-20_08:50:57-upgrade-master---basic-mira/

**Comment by tchaikov** [comment]
```
2017-06-20T11:08:05.455 INFO:tasks.workunit.client.1.mira071.stdout:                 api_misc--gtest_filter=-*WriteSame*:-*CmpExt*:-*Checksum*: [  FAILED  ] 2 tests, listed below:
2017-06-20T11:08:05.455 INFO:tasks.workunit.client.1.mira071.stdout:                 api_misc--gtest_filter=-*WriteSame*:-*CmpExt*:-*Checksum*: [  FAILED  ] LibRadosMisc.CloneRange
2017-06-20T11:08:05.455 INFO:tasks.workunit.client.1.mira071.stdout:                 api_misc--gtest_filter=-*WriteSame*:-*CmpExt*:-*Checksum*: [  FAILED  ] LibRadosMiscPP.CloneRangePP
2017-06-20T11:08:05.455 INFO:tasks.workunit.client.1.mira071.stdout:                 api_misc--gtest_filter=-*WriteSame*:-*CmpExt*:-*Checksum*:
2017-06-20T11:08:05.455 INFO:tasks.workunit.client.1.mira071.stdout:                 api_misc--gtest_filter=-*WriteSame*:-*CmpExt*:-*Checksum*:  2 FAILED TESTS
```

**Comment by tchaikov** [comment]
retest this please.

**Comment by yuriw** [comment]
@tchaikov do we want to add it here or to a new luminous suites ?

**Comment by tchaikov** [comment]
@yuriw, i guess once luminous is released and start working on mimic, we'd need `hammer-jewel-luminous` upgrade suite. @liewegas what do you think?

File: qa/workunits/rbd/import_export.sh [hunk 1]
```diff
@@ -74,70 +74,72 @@ cmp ${TMPDIR}/img ${TMPDIR}/img3
...
-# features
-rbd import --image-feature layering ${TMPDIR}/img testimg
-FEATURES_BEFORE=`rbd info testimg|grep features`
-rbd export --export-format 2 testimg ${TMPDIR}/img_v2
-rbd import --export-format 2 ${TMPDIR}/img_v2 testimg_import
-FEATURES_AFTER=`rbd info testimg_import|grep features`
-if [ "$FEATURES_BEFORE" != "$FEATURES_AFTER" ]; then
-  false
-fi
+if rbd help export | grep -q export-format; then
```
Comment (tchaikov) [inline qa/workunits/rbd/import_export.sh]: this change fixes the failure of http://pulpito.ceph.com/kchai-2017-06-22_00:11:35-upgrade-master---basic-mira/

@dillaman could you help review this commit?

**Pull request merged**